DocGen4/Output/DocString.lean

```diff
@@ -55,9 +54,7 @@ partial def xmlGetHeadingId (el : Xml.Element) : String :=
       |>.filter (!·.isEmpty)
       |> replacement.intercalate
     unicodeToDrop (c : Char) : Bool :=
-      charInGeneralCategory c GeneralCategory.punctuation ||
-      charInGeneralCategory c GeneralCategory.separator ||
-      charInGeneralCategory c GeneralCategory.other
+      false -- TODO: restore the behavior described in the docstring
```

Comment:
`c == " "` is probably a marginally better choice

Reply:
Sure, go for it then :+1: